common/src/main/java/com/graphicsfuzz/common/util/FragmentShaderToShaderJob.java

```diff
@@ -114,12 +114,21 @@ public static ShaderJob createShaderJob(TranslationUnit tu, IRandom generator) {
     // TranslationUnit with corresponding 'out' variables and add it to 'shaders'.
     Optional<TranslationUnit> vertexShader = Optional.empty();
 
+    // Start layout locations from index 1
+    int locationIndex = 1;
+
     for (VariablesDeclaration vd : tu.getGlobalVariablesDeclarations()) {
       if (vd.getBaseType().hasQualifier(TypeQualifier.SHADER_INPUT)) {
         final BasicType basicType = (BasicType) vd.getBaseType().getWithoutQualifiers();
         // Create clone of the variables declaration and change the input to output:
         final QualifiedType varType = (QualifiedType) vd.getBaseType().clone();
         varType.replaceQualifier(TypeQualifier.SHADER_INPUT, TypeQualifier.SHADER_OUTPUT);
+
+        // Set same layout location for vertex output and fragment input
+        ((QualifiedType)vd.getBaseType()).setLocationQualifier(locationIndex);
```

Comment:
Google style requires space after closing ')' of a cast, so '(QualifiedType) vd' here.  (I guess this is a somewhat hard thing for CheckStyle to enforce.

Reply:
done